src/commands/InitCommand.ts

```diff
@@ -601,6 +601,7 @@ Steps to run this project:
         if (!packageJson.scripts) packageJson.scripts = {};
         Object.assign(packageJson.scripts, {
             start: /*(docker ? "docker-compose up && " : "") + */"ts-node src/index.ts"
+            , typeorm: "node --require ts-node/register ./node_modules/typeorm/cli.js"
```

Comment:
Can you place the comma on the previous line to match the style we traditionally use

Reply:
Thank you for your review. I modified the comma style and committed.

Reply:
@imnotjames 
Thank you for your review. I modified the comma style and committed.